mindmap/static/js/src/mindmap.js

```diff
@@ -1,39 +1,90 @@
 // TODO: add notifications
 function MindMapXBlock(runtime, element, context) {
 
-    const baseMind = {
-        "meta": {
-            "name": "Mind Map",
-            "version": "0.1"
-        },
-        "format": "node_array",
-        "data": [
-            { "id": "root", "isroot": true, "topic": "Root" },
-        ]
-    };
+    function showMindMap(jsMind, context) {
+        const baseMind = {
+            meta: {
+                name: "Mind Map",
+                version: "0.1",
+            },
+            format: "node_array",
+            data: [{ id: "root", isroot: true, topic: "Root" }],
+        };
 
-    const mind = context.mind_map || baseMind;
-    mind.meta.author = context.author;
+        const mind = context.mind_map || baseMind;
+        mind.meta.author = context.author;
 
-    const options = {
-        container: 'jsmind_container',
-        editable: true,
-        theme: 'asphalt',
-    };
+        const options = {
+            container: "jsmind_container",
+            editable: context.editable,
+            theme: "asphalt",
```

Comment:
[request for another PR]: how many other themes are there?

Reply:
There are currently 15 themes according to [the documentation](https://github.com/hizzgdev/jsmind/blob/master/docs/en/1.usage.md#13-themes)

Reply:
I opened a new Jira issue to track this!